README.md

```diff
@@ -1,54 +1,76 @@
+
 * [**examples/SimpleStorageWriteRead**](https://github.com/arduino-libraries/Arduino_UnifiedStorage/blob/main/examples/SimpleStorageWriteRead/SimpleStorageWriteRead.ino) - Write/read simple data from SD, USB and internal storage
+
 * [**examples/AdvancedUSBInternalOperations**](https://github.com/arduino-libraries/Arduino_UnifiedStorage/blob/main/examples/AdvancedUSBInternalOperations/AdvancedUSBInternalOperations.ino) - Navigate file structure and demonstrate file operations between USB and internal storage
+
 * [**examples/BackupInternalPartitions**](https://github.com/arduino-libraries/Arduino_UnifiedStorage/blob/main/examples/BackupInternalPartitions/BackupInternalPartitions.ino) - Back up all partitions on the internal storage to a USB Mass Storage device.
 
+* [**examples/Logger**](https://github.com/arduino-libraries/Arduino_UnifiedStorage/blob/main/examples/Logger/Logger.ino) - This example demonstrates how you can log sensor data into its internal storage and detect the insertion of a USB drive. When a USB drive is connected, it selectively transfers only the new data to the USB mass storage device. Even if you remove the USB drive, the device continues to log data in its internal storage. This capability is particularly valuable for accumulating training data required for embedded machine learning applications.
+ 
+ * [**examples/InternalStoragePartitioning**](https://github.com/arduino-libraries/Arduino_UnifiedStorage/blob/main/examples/InternalStoragePartitioning/InternalStoragePartitioning.ino) - Demonstrates how you can use the Arduino_UnifiedStorage library to create and read partitions on the Internal QSPI Storage. 
+ 
 ## 👀 Instructions
-1. Download and install this library 
+
+1. Download and install this library
 2. Check compatibility with your platform
-3. To use internal storage, you need to make sure it is partitioned and formatted correctly:
-
-### Formatting the Portenta H7 / Portenta Machine Control / Opta Internal Storage
-* Flash the `QSPIFormat` example that can be found in the `STM32H747_System` folder
-* Open the serial monitor and select answer with "n" when this appears "Do you want to use partition scheme 1? Y/[n]"
-* The sketch will warn you that the content of the QSPI flash will be erased. Answer with "Y".
-* When asked if you'd like to use LittleFS on the data partition, select "n". Most of the examples assume that the drive is formatted as FAT. You can use the library to format to LittleFS later. 
-* Reboot the board
- 
-### Formatting the Portenta C33 Internal Storage
-*  Flash the `QSPIFormat` example that can be found in the `Storage` folder for Portenta C33.
-*  Open the Serial Monitor. The sketch will warn you that the content of the QSPI flash will be erased. Answer with "Y".
-*  When asked if you'd like to use LittleFS on the data partition, select "n". Most of the examples assume that the drive is formatted as FAT. You can use the library to format to LittleFS later. 
-*  Reboot the board
```

Comment:
The formatting steps are not required anymore for internal storage, correct?

Reply:
correct